aries-site/src/examples/components/page/PageContentExample.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Box,
+  Card,
+  Grid,
+  Heading,
+  Paragraph,
+  Page,
+  PageContent,
+} from 'grommet';
+
+export const PageContentExample = () => (
+  <Page kind="narrow">
```

Comment:
I think since this example looks like a dashboard we should use `kind="wide"` here. Narrow looks a little strange for this example and doesn't fit with the use cases listed for narrow on the Page Layouts guidance.